### PR TITLE
Version 1.27.0 → 1.28.0

### PR DESCRIPTION
**Pull request by Gabriel439**

### CODE REVIEW

File: dhall-yaml/dhall-yaml.cabal [hunk 1]
```diff
@@ -37,8 +37,8 @@ Library
         base                      >= 4.8.0.0  && < 5   ,
         aeson                     >= 1.0.0.0  && < 1.5 ,
         bytestring                               < 0.11,
-        dhall                     >= 1.27.0   && < 1.28,
-        dhall-json                >= 1.5.0    && < 1.6 ,
+        dhall                     >= 1.28.0   && < 1.29,
+        dhall-json                >= 1.5.0    && < 1.7 ,
```
Comment (sjakobi) [inline dhall-yaml/dhall-yaml.cabal]: `dhall-yaml` relies on some changes that I implemented as a part of the package split.

```suggestion
        dhall-json                >= 1.6.0    && < 1.7 ,
```

File: dhall-json/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+        * The YAML-related modules in this package were renamed
+        * This package no longer accepts a `-fgpl` cabal configure flag
+        * This package no longer provides the `yaml-to-dhall` executable (which
+          now resides within the `dhall-yaml` package)
+* [Don't normalize schema before type-checking](https://github.com/dhall-lang/dhall-haskell/pull/1555)
+    * This improves the error messages for `{json,yaml}-to-dhall`
+* [Fix `dhall-to-yaml` to quote special strings](https://github.com/dhall-lang/dhall-haskell/pull/1474)
+* [Make some of CLI options for `dhall-json` more consistent](https://github.com/dhall-lang/dhall-haskell/pull/1475)
+
+1.5.0
```
Comment (sjakobi) [inline dhall-json/CHANGELOG.md]: The renaming below belongs to the new release.

**Comment by Gabriel439** [comment]
I'll wait a couple of days before merging, as usual.  If #1561 does not get in before then we can still cut a 1.28.1 release afterwards once it is ready.

**Comment by sjakobi** [comment]
Sorry for the untimely flood of PRs @Gabriel439! Feel free to merge what you'd like to include in this release – I don't really mind. :)

**Pull request merged**